tests/framework/helpers/ConsoleTest.php

```diff
@@ -129,4 +129,45 @@ public function testAnsi2Html($ansi, $html)
     {
         $this->assertEquals($html, Console::ansiToHtml($ansi));
     }
+
+    public function testTable()
+    {
+        $expected = <<<EXPECTED
++---------------+---------------+---------------+
+| test1         | test2         | test3         |
++---------------+---------------+---------------+
+| testcontent1  | testcontent2  | testcontent3  |
++---------------+---------------+---------------+
+| testcontent21 | testcontent22 | testcontent23 |
++---------------+---------------+---------------+
+
+EXPECTED;
+
+        $this->assertEqualsWithoutLE($expected, Console::table(
+            ['test1', 'test2', 'test3'],
+            [
+                ['testcontent1', 'testcontent2', 'testcontent3'],
+                ['testcontent21', 'testcontent22', 'testcontent23']
+            ]
+        ));
+
+        $expected = <<<EXPECTED
+*++++++++++++++++*+++++++++++++++++*++++++++++++++++++*
+/ test1          / test2           / test3            /
```

Comment:
Also it should be possible to format nicer: https://www.npmjs.com/package/cli-table2